examples/features/nse-composition/README.md

```diff
@@ -112,7 +112,7 @@ kubectl create -f ./nse-composition-ns.yaml
 
 Deploy NSC and NSE:
 ```bash
-kubectl apply -k .
+kustomize build . | kubectl apply -f -
```

Comment:
Please don't use `kustomize` directly. All kustomize operations should be done naturally via `kubectl apply -k`

Reply:
This will be fixed in https://github.com/networkservicemesh/deployments-k8s/issues/2123